source/geometry_model/chunk.cc

```diff
@@ -707,17 +747,30 @@ namespace aspect
                                    "chunk",
                                    "A geometry which can be described as a chunk of a spherical shell, "
                                    "bounded by lines of longitude, latitude and radius. "
-                                   "The minimum and maximum longitude, (latitude) and depth of the chunk "
+                                   "The minimum and maximum longitude, latitude (if in 3d) and depth of the chunk "
                                    "is set in the parameter file. The chunk geometry labels its "
                                    "2*dim sides as follows: ``west'' and ``east'': minimum and maximum "
                                    "longitude, ``south'' and ``north'': minimum and maximum latitude, "
                                    "``inner'' and ``outer'': minimum and maximum radii. "
-                                   "Names in the parameter files are as follows: "
+                                   "\n\n"
+                                   "The dimensions of the model are specified by parameters "
+                                   "of the following form: "
                                    "Chunk (minimum || maximum) (longitude || latitude): "
                                    "edges of geographical quadrangle (in degrees)"
                                    "Chunk (inner || outer) radius: Radii at bottom and top of chunk"
                                    "(Longitude || Latitude || Radius) repetitions: "
-                                   "number of cells in each coordinate direction.")
+                                   "number of cells in each coordinate direction."
+                                   "\n\n"
+                                   "When used in 2d, this geometry does not imply the use of "
+                                   "a spherical coordinate system. Indeed, "
+                                   "in 2d the geometry is simply a sector of an annulus in a Cartesian "
+                                   "coordinate system and consequently would correspond to "
```

Comment:
I am rereading this, and I think the statement that we use a Cartesian coordinate system also holds for 3d geometry models. Although we have some wrappers which pretend everything is spherical, all the geometry models work in the end through Cartesian coordinates. Should we add that, or am I misinterpreting the text?

Reply:
Ok. Yes, I noticed this came from the mailing list :)